src/components/WorkspaceMenu.js

```diff
@@ -153,6 +157,16 @@ export class WorkspaceMenu extends Component {
             </ListItemIcon>
             <Typography varient="inherit">{t('downloadExportWorkspace')}</Typography>
           </MenuItem>
+          <MenuItem
+            aria-haspopup="true"
+            onClick={(e) => { this.handleMenuItemClick('importWorkspace', e); handleClose(e); }}
+            aria-owns={exportWorkspace.AnchorEl ? 'workspace-import' : undefined}
+          >
+            <ListItemIcon>
+              <ImportIcon />
+            </ListItemIcon>
+            <Typography varient="inherit">{t('importWorkspace')}</Typography>
```

Comment:
If you don't discover this due to merge conflicts, I believe this should be `variant="body"` per the new typography specs (note the typo in the key too; we've copy+pasted that all over)